Library/ViewModels/SearchViewModelTests.swift

```diff
@@ -684,7 +684,7 @@ internal final class SearchViewModelTests: TestCase {
 
         self.scrollToProjectRow.assertValues([5, 6, 7, 8])
 
-        self.hasAddedProjects.assertValues([true, false, true, true], "Paginated projects are loaded.")
```

Comment:
I know I said no regressions but I'm open to guesses as to why this test needed to change 😅. I can't see any reason that the change I made would cause this 🤔

Reply:
Appears there is another failing snapshot test so will investigate.

Reply:
Not sure if @tobitech may know what's going on here? I remember him working on the stats piece. Feel free to disregard if you don't know.